src/models/FilterGrower.js

```diff
@@ -55,6 +56,24 @@ export default class Filter {
       where.phone = this.phone;
     }
 
+    if (
+      !isNaN(this.capturesAmount_min) ||
+      (this.capturesAmount_range && !isNaN(this.capturesAmount_range.min))
+    ) {
+      where.capturesAmount_min = !isNaN(this.capturesAmount_min)
+        ? this.capturesAmount_min
+        : this.capturesAmount_range.min;
+    }
+
+    if (
+      !isNaN(this.capturesAmount_max) ||
+      (this.capturesAmount_range && !isNaN(this.capturesAmount_range.max))
+    ) {
+      where.capturesAmount_max = !isNaN(this.capturesAmount_max)
+        ? this.capturesAmount_max
+        : this.capturesAmount_range.max;
+    }
+
```

Comment:
// Problem: this method is called twice.
        //    1st call (from GrowerContext.js - load()): 'this' is an object with 'capturesAmount_range' property. Return object has property 'capturesAmount_min'
       //    2nd call (from growers.js - getGrowers()): 'this' is an object with 'capturesAmount_min' property.
// Workaround: initialize 'where' object with properties which are available
// Better solution: make the only one call

// For the same reason 'firstName' and 'lastName' filters don't work at the moment

Reply:
I think you're right – `GrowerContext.js` shouldn't call `getWhereObj()` directly, since this returns the object used the query the API. We're effectively converting the object twice.

Instead, I think `setOrganizationFilter()`should modify a copy of the `FilterGrower` object.

As an aside, we're doing the same thing in `VerifyContext.js` and `CapturesContext.js` – we'll need to apply the same fix there.

Well spotted 😃

Reply:
There are a some other filtering bugs that _may_ be related to this: #1016, #1034, #1035
@gwynndp: Can you coordinate with @OlhaD the fix for this works coherently across the different filters?
I'm happy for everything to be fixed under this PR, if it's all caused by the same thing.

Reply:
@nmcharlton I looked at all 3 issues you mentioned and seems that they are not related to this one.